GoogleChartBehavior.js

```diff
@@ -0,0 +1,264 @@
+   * @constructor
+   * @param options {Object} configuration for this behavior.
+   *   @param [options._getChartType=this._getChartType] {Function} See GoogleChartBehavior._getChartType for details.
+   *   @param [options._chartOptions=this._chartOptions] {Object|Function} See GoogleChartBehavior._chartOptions for details.
+   *   @param [options._chartContainerSelector=this._chartContainerSelector] {String|Function} See GoogleChartBehavior._chartContainerSelector for details.
+   *   @param [options._createChartDataSource=this._createChartDataSource] {Function} See GoogleChartBehavior._createChartDataSource for details.
+   *   @param [options._initializeChartDataSource=this._initializeChartDataSource] {Function} see GoogleChartBehavior._initializeChartDataSource for details.
+   *   @param [options._resetChartDataSource=this._resetChartDataSource] {Function} see GoogleChartBehavior._resetChartDataSource for details.
+   *   @param [options._fillChartDataSource=this._fillChartDataSource] {Function} see GoogleChartBehavior._fillChartDataSource for details.
+   *   @param [options._getChartDataSourceRows=this._getChartDataSourceRows] {Function} see GoogleChartBehavior._getChartDataSourceRows for details.
+   * @author jyoung
+   */
+  Torso.Extensions.GoogleChartBehavior = Torso.Behavior.extend({
+    /**
+     * A function that will return the constructor to use when creating the google chart.
+     * Deferred to a function so that it can be run after google charts has loaded.
+     * @method _getChartType
+     * @return {Function} a constructor for a google chart.  Default value is google.visualization.LineChart
+     * @private
+     */
+    _getChartType: function() {
+      return google.visualization.LineChart;
+    },
+
+    /**
+     * The configuration to use for the google chart or a function that returns the options to use.  The function is scoped to this behavior.
+     * @property _chartOptions {Object|Function}
+     * @default null
+     * @private
+     */
+    _chartOptions: null,
+
+    /**
+     * The jQuery selector to use to identify the DOM element owned by this view that will contain the chart.
+     * The container element will be ignored by template renderer - it will only be updated by the chart's draw function.
+     * Can be the selector as a string or a function that returns the jQuery selector string.  The function will be scoped to this behavior.
+     * @property _chartContainerSelector {String|Function}
+     * @default '[data-container=chart]'
+     * @private
+     */
+    _chartContainerSelector: '[data-chart-container]', 
+
+    /**
+     * @property mixin {Object}
+     * @override
+     */
+    mixin: {
+      /**
+       * Overrides the view's templateRendererOptions to configure the renderer to ignore the chart containers.
+       * @method templateRendererOptions
+       * @return {Object} the options to pass to the template renderer.
+       */
+      templateRendererOptions: function() {
```

Comment:
this is going to be hard to stack with other behaviors that utilize this, but we knew that was the case.